docker/contracts/Dockerfile

```diff
@@ -25,6 +25,7 @@ RUN cargo install wasm-opt --locked
 # Clone the contracts repo
 WORKDIR /sources
 RUN git clone \
+    -b andrew/srs-file \
```

Comment:
Note: this will be removed when https://github.com/renegade-fi/renegade-contracts/pull/153 is merged

Reply:
approve that PR, bumping this so it doesn't get overlooked before merge